src/dispatch/auth/permissions.py

```diff
@@ -291,3 +293,71 @@ def has_required_permissions(
         if current_incident.commander:
             if current_incident.commander.individual.email == current_user.email:
                 return True
+
+
+# Cases
+
+
+class CaseViewPermission(BasePermission):
+    def has_required_permissions(
+        self,
+        request: Request,
+    ) -> bool:
+        current_case = case_service.get(
+            db_session=request.state.db, case_id=request.path_params["case_id"]
+        )
+
+        if not current_case:
+            return False
+
+        if current_case.visibility == Visibility.restricted:
```

Comment:
Shouldn't any case participant have access to view the case?

Reply:
I think so, I'll add another permission that handles this for all participants (and keep the existing reporter/assignee ones in case we want to keep those to have different permission models.)

Reply:
Scratch that, I just switched it to be "if you're a participant, you can view the case, and edit it."